x-pack/plugins/code/public/monaco/definition/definition_provider.ts

```diff
@@ -12,7 +12,7 @@ import { Location } from 'vscode-languageserver-types';
 import { LspRestClient, TextDocumentMethods } from '../../../common/lsp_client';
 
 export function provideDefinition(monaco: any, model: editor.ITextModel, position: any) {
-  const lspClient = new LspRestClient('../api/lsp');
+  const lspClient = new LspRestClient('/api/code/lsp');
```

Comment:
so this should fix an existing bug. or the `/api/lsp` route is also a valid endpoint?